### ISSUE TITLE
Dodgy parse error when expected token is at the end of a line

### ISSUE THREAD

**Issue by jmackie** [ditto-lang/ditto#34]
Compiling the following module:

```ditto
module Issue34 exports (..)

huh = 42;
```

Prints:
```
  × syntax error
   ╭─[src/Test.ditto:2:1]
 2 │
   ·  ▲
   ·  ╰── expected: semicolon
 3 │ huh = 42;
   · ▲
   · ╰── expected: semicolon
   ╰────
```

But I would expect: 

```
  × syntax error
   ╭─[src/Test.ditto:1:1]
 1 │ module Issue34 exports (..)
   ·                              ▲
   ·                              ╰── expected: semicolon
   ╰────
```

This is a known issue in [`miette`](https://github.com/zkat/miette) (zkat/miette#130) so we need to look at pushing a fix there.

**Comment by zkat** [comment]
This is fixed as of miette@4.6.0. Go ahead and bump your version and you should be good to go :)

**Comment by jmackie** [comment]
Amazing, can confirm this is fixed by #30 🎉 

Thanks very much for fixing! 🙏
